AutomatedTesting/Gem/PythonTests/scripting/ScriptEvents_AllParamDatatypes_CreationSuccess.py

```diff
@@ -92,27 +91,18 @@ async def run_test(self):
         # 1) Open Asset Editor
         # Initially close the Asset Editor and then reopen to ensure we don't have any existing assets open
         general.close_pane(ASSET_EDITOR_UI) # this doesn't close a file that was previously open if you had just run a test that created an asset
-        general.open_pane(ASSET_EDITOR_UI)
-        helper.wait_for_condition(lambda: general.is_pane_visible(ASSET_EDITOR_UI), WAIT_TIME_3)
-
-        # 2) Initially create new Script Event file with one method
-        tools.initialize_asset_editor_qt_objects(self)
-        action = pyside_utils.find_child_by_pattern(self.asset_editor_menu_bar, {"type": QtWidgets.QAction, "text": SCRIPT_EVENT_UI})
-        action.trigger()
-        result = helper.wait_for_condition(
-            lambda: self.asset_editor_row_container.findChild(QtWidgets.QFrame, EVENTS_QT) is not None
-                    and self.asset_editor_row_container.findChild(QtWidgets.QFrame, EVENTS_QT).findChild(QtWidgets.QToolButton, "") is not None,
-            WAIT_TIME_3,
-        )
-        Report.result(Tests.new_event_created, result)
+        result = tools.open_asset_editor()
+        Report.result(Tests.asset_editor_opened, result)
+        tools.initialize_editor_object(self)
+
+        # 2) Initialize asset editor, script canvas and create new Script Event file with one method
+        tools.initialize_asset_editor_object(self)
+        result = tools.open_script_canvas()
+        Report.result(Tests.script_canvas_opened, result)
```

Comment:
NIT: You can inline the result in the report call to reduce the code and a few steps.

Reply:
I think I will leave these as they are for readability. I don't struggle with in-line so much but other may and we definitely have to return to these in time.